github/src/main/kotlin/gropius/sync/github/model/TimelineEventInfo.kt

```diff
@@ -4,10 +4,12 @@ import org.bson.types.ObjectId
 import org.springframework.data.annotation.Id
 import org.springframework.data.mongodb.core.index.Indexed
 import org.springframework.data.mongodb.core.mapping.Document
+import java.net.URI
 import java.time.OffsetDateTime
 
 /**
  * Mapping of a single timeline event from neo4j to github
+ * @param url API URL of IMS of the repo
```

Comment:
consistent documentation in same class

Reply:
f22dc76eecf06c31cba4638560eb8dafa04cf6c7